src/transformers/models/ctrl/modeling_tf_ctrl.py

```diff
@@ -571,39 +571,26 @@ def serving_output(self, output):
         )
 
 
-class TFCTRLLMHead(tf.keras.layers.Layer):
-    def __init__(self, config, input_embeddings, **kwargs):
-        super().__init__(**kwargs)
-        self.config = config
-        # CTRL has numerical issues in XLA generate
-        self.supports_xla_generation = False
+class BiasLayer(tf.keras.layers.Layer):
```

Comment:
Slightly different than the separate LM bias head for BART, because of the serialization name. In BART, the bias weights are at root level (and thus added in `__init__`). Here, we want it nested as `lm_head/bias:0`, so it gets initialized in `build()`.